docs/API/SandboxSetup.md

```diff
@@ -4,48 +4,52 @@ Chaincode developers need a way to test and debug their chaincode without having
 
-###Vagrant Terminal 1 (validating peer)
-**Note:** To run with security enabled, first modify the [core.yaml](https://github.com/hyperledger/fabric/blob/master/peer/core.yaml) configuration file to set the `security.enabled` value to `true` before building the peer executable. Alternatively, you can enable security by running the peer with environment variable `CORE_SECURITY_ENABLED=true`. To enable privacy and confidentiality of transactions (requires security to also be enabled), modify the [core.yaml](https://github.com/hyperledger/fabric/blob/master/peer/core.yaml) configuration file to set the `security.privacy` value to `true` as well. Alternatively, you can enable privacy by running the peer with environment variable `CORE_SECURITY_PRIVACY=true`. If you are enabling security and privacy on the peer process with environment variables, it is important to include these environment variables in the command when executing all subsequent peer operations (e.g. deploy, invoke, or query).
+## Vagrant Terminal 1 (validating peer)
+> **Note:** To run with security enabled, first modify the [core.yaml](https://github.com/hyperledger/fabric/blob/master/peer/core.yaml) configuration file to set the `security.enabled` value to `true` before building the peer executable. Alternatively, you can enable security by running the peer with environment variable `CORE_SECURITY_ENABLED=true`. To enable privacy and confidentiality of transactions (requires security to also be enabled), modify the [core.yaml](https://github.com/hyperledger/fabric/blob/master/peer/core.yaml) configuration file to set the `security.privacy` value to `true` as well. Alternatively, you can enable privacy by running the peer with environment variable `CORE_SECURITY_PRIVACY=true`. If you are enabling security and privacy on the peer process with environment variables, it is important to include these environment variables in the command when executing all subsequent peer operations (e.g. deploy, invoke, or query).
 
 From your command line terminal, move to the `devenv` subdirectory of your workspace environment. Log into a Vagrant terminal by executing the following command:
 
     vagrant ssh
 
-Build and run the peer process to enable security and privacy after setting `security.enabled` and `security.privacy` settings to `true`.
+Build and run the peer process:
 
     cd $GOPATH/src/github.com/hyperledger/fabric
     make peer
-    peer node start --peer-chaincodedev
+    cd build/bin
```

Comment:
It seems the PATH is not set in my vagrant VM. That’s why I never noticed this mechanism.

vagrant@ubuntu-1404:/hyperledger$ echo $PATH
/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/opt/go/bin:/opt/gopath/bin



> On 14 Jul 2016, at 11:50, Sheehan Anderson <notifications@github.com> wrote:
> 
> In docs/API/SandboxSetup.md <https://github.com/hyperledger/fabric/pull/2204#discussion_r70745084>:
> 
> >  
> >      cd $GOPATH/src/github.com/hyperledger/fabric
> >      make peer
> > -    peer node start --peer-chaincodedev
> > +    cd build/bin
> Something likely failed during the vagrant up process. The build/bin folder should have been put on your PATH. See https://github.com/hyperledger/fabric/blob/master/devenv/setup.sh#L110 <https://github.com/hyperledger/fabric/blob/master/devenv/setup.sh#L110>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/hyperledger/fabric/pull/2204/files/5716e2bd3e3574a292a245408f448fe120dc89fc#r70745084>, or mute the thread <https://github.com/notifications/unsubscribe/AAYwiT07lSLE_XCpMQXEeHkOXjPQ5Rnhks5qVbH9gaJpZM4JMDBv>.
>